src/main/java/cz/kominekjan/disenchantment/commands/impl/Enchantments.java

```diff
@@ -9,70 +12,110 @@
 
 import java.util.Map;
 
-import static cz.kominekjan.disenchantment.config.Config.getDisabledEnchantments;
-import static cz.kominekjan.disenchantment.config.Config.setDisabledEnchantments;
 import static cz.kominekjan.disenchantment.utils.TextUtils.*;
 
-public class Enchantments {
-    public static final Command command = new Command(
+public abstract class Enchantments implements ICommandExecutor {
+    public static final Command disenchantCommand = new Command(
             "enchantments",
             new String[]{"disenchantment.all", "disenchantment.command.enchantments"},
             "You don't have permission to use this command.",
             new String[]{},
             false,
-            Enchantments::execute
+            new Enchantments() {
+                @Override
+                public Map<Enchantment, EnchantmentStatus> getEnchantmentsStatus() {
+                    return Config.getEnchantmentsStatus();
+                }
+
+                @Override
+                public void setEnchantmentsStatus(Enchantment enchantment, EnchantmentStatus status) {
+                    Config.setEnchantmentsStatus(enchantment, status);
+                }
+            }
+    );
+    public static final Command bookSplittingCommand = new Command(
+            "split-enchantments",
+            new String[]{"disenchantment.all", "disenchantment.command.enchantments"},
+            "You don't have permission to use this command.",
+            new String[]{},
+            false,
+            new Enchantments() {
+                @Override
+                public Map<Enchantment, EnchantmentStatus> getEnchantmentsStatus() {
+                    return Config.getBookSplittingEnchantmentsStatus();
+                }
+
+                @Override
+                public void setEnchantmentsStatus(Enchantment enchantment, EnchantmentStatus status) {
+                    Config.setBookSplittingEnchantmentsStatus(enchantment, status);
+                }
+            }
```

Comment:
really not big fan of this, as mentioned in another comment, please divide it into two files that are easier to read and does not involve abstract methods, thanks

Reply:
Resolved on https://github.com/H7KZ/Disenchantment/pull/15/commits/be329a3a35cfd14a12c38932cd9332b56b2ba090